src/uct/ib/rc/base/rc_ep.c

```diff
@@ -20,29 +20,31 @@
 
 #ifdef ENABLE_STATS
 static ucs_stats_class_t uct_rc_fc_stats_class = {
-    .name = "rc_fc",
+    .name         = "rc_fc",
     .num_counters = UCT_RC_FC_STAT_LAST,
-    .counter_names = {
-        [UCT_RC_FC_STAT_NO_CRED]            = "no_cred",
-        [UCT_RC_FC_STAT_TX_GRANT]           = "tx_grant",
-        [UCT_RC_FC_STAT_TX_PURE_GRANT]      = "tx_pure_grant",
-        [UCT_RC_FC_STAT_TX_SOFT_REQ]        = "tx_soft_req",
-        [UCT_RC_FC_STAT_TX_HARD_REQ]        = "tx_hard_req",
-        [UCT_RC_FC_STAT_RX_GRANT]           = "rx_grant",
-        [UCT_RC_FC_STAT_RX_PURE_GRANT]      = "rx_pure_grant",
-        [UCT_RC_FC_STAT_RX_SOFT_REQ]        = "rx_soft_req",
-        [UCT_RC_FC_STAT_RX_HARD_REQ]        = "rx_hard_req",
-        [UCT_RC_FC_STAT_FC_WND]             = "fc_wnd"
-    }
+    .class_id     = 0,
+    .counter_names =
+            {
+                    [UCT_RC_FC_STAT_NO_CRED]       = "no_cred",
+                    [UCT_RC_FC_STAT_TX_GRANT]      = "tx_grant",
+                    [UCT_RC_FC_STAT_TX_PURE_GRANT] = "tx_pure_grant",
+                    [UCT_RC_FC_STAT_TX_SOFT_REQ]   = "tx_soft_req",
+                    [UCT_RC_FC_STAT_TX_HARD_REQ]   = "tx_hard_req",
+                    [UCT_RC_FC_STAT_RX_GRANT]      = "rx_grant",
+                    [UCT_RC_FC_STAT_RX_PURE_GRANT] = "rx_pure_grant",
+                    [UCT_RC_FC_STAT_RX_SOFT_REQ]   = "rx_soft_req",
+                    [UCT_RC_FC_STAT_RX_HARD_REQ]   = "rx_hard_req",
+                    [UCT_RC_FC_STAT_FC_WND]        = "fc_wnd"}
 };
 
 static ucs_stats_class_t uct_rc_txqp_stats_class = {
-    .name = "rc_txqp",
+    .name         = "rc_txqp",
     .num_counters = UCT_RC_TXQP_STAT_LAST,
-    .counter_names = {
-        [UCT_RC_TXQP_STAT_QP_FULL]          = "qp_full",
-        [UCT_RC_TXQP_STAT_SIGNAL]           = "signal"
-    }
+    .class_id     = 0,
+    .counter_names =
```

Comment:
plz align by `=`

Reply:
pls revert prev formatting of "counter_names" block